src/zenml/zen_server/routers/auth_endpoints.py

```diff
@@ -255,7 +255,7 @@ def generate_access_token(
     LOGIN,
     response_model=Union[OAuthTokenResponse, OAuthRedirectResponse],
 )
-@handle_exceptions
+# @handle_exceptions
```

Comment:
WTH is this for ?

Reply:
... i'm guessing a bird flew into my room when I wasn't looking and landed on the # key. I don't see any other explanation.